packages/dbos-cloud/cli.ts

```diff
@@ -139,9 +139,9 @@ const databaseCommands = program
 databaseCommands
   .command('provision')
   .description("Provision a Postgres database instance")
-  .argument('<string>', 'database instance name')
-  .requiredOption('-a, --admin <string>', 'Specify the admin user')
-  .requiredOption('-W, --password <string>', 'Specify the admin password')
+  .argument('<name>', 'database instance name')
```

Comment:
What is the best practice for these hints? Data type or expected logical value? (We probably want to harmonize across the commands.)

Reply:
I think they should be the name, otherwise, you'll see:
```
Usage: dbos-cloud database reset-password [options] <string>    
```
which looks more confusing.

Reply:
We can have another pass to beautify CLI UI